### PR TITLE
Managed contacts

### PR DESCRIPTION
**Pull request by Dorthu**
This is a migrated https://github.com/linode/manager/pull/2674

### CODE REVIEW

File: src/data/endpoints/managed.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          Registers a new contact to your managed service contacts list.
+        params:
+          name:
+            description: The contact's full name.
+            type: String
+            limit: "1-64 characters"
+          email:
+            description: The email address for the contact.
+            type: String
+          phone1:
```
Comment (yoshizzle) [inline src/data/endpoints/managed.yaml]: should be "primary" and in a "phone" envelope

File: src/data/endpoints/managed.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            description: A primary phone number for the contact.
+            type: String
+          phone2:
```
Comment (yoshizzle) [inline src/data/endpoints/managed.yaml]: should be "secondary" and in a "phone" envelope

File: src/data/endpoints/managed.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            description: Contacts can be associated with a monitored service by the group they belong to.
+            type: String
+        examples:
+          curl: |
+            curl -H "Content-Type: application/json" \
+                -H "Authorization: Bearer $TOKEN" \
+                -X POST -d '{
+                    "name": "John Q. Public",
+                    "email": "johnqpublic@linode.com",
+                    "phone1": "215-555-3000",
```
Comment (yoshizzle) [inline src/data/endpoints/managed.yaml]: "phone": {
    "primary": "215-555-3000",
    "secondary": "215-555-3001"
}

File: src/data/endpoints/managed.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        description: >
+          Edits this contact.
+        examples:
+          curl: |
+            curl -H "Content-Type: application/json" \
+                -H "Authorization: Bearer $TOKEN" \
+                -X PUT -d '{
+                    "name": "John Q. Public, Jr.",
+                    "email": "johnqpublicjr@linode.com",
+                    "phone1": "215-555-4000",
```
Comment (yoshizzle) [inline src/data/endpoints/managed.yaml]: same as above

File: src/data/objects/managedcontact.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    value: John Q. Public
+    type: String
+    description: >
+      The name of the contact person.
+  email:
+    value: jsmith@mycompany.com
+    type: String
+    description: >
+      The email address of the contact person.
+  phone1:
```
Comment (yoshizzle) [inline src/data/objects/managedcontact.yaml]: primary

File: src/data/objects/managedcontact.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    value: 215-555-3000
+    type: String
+    description: >
+      If we need further information regarding a service, we may call this contact if a phone number is provided.
+  phone2:
```
Comment (yoshizzle) [inline src/data/objects/managedcontact.yaml]: secondary

File: src/data/endpoints/managed.yaml (hunk 1 above)
Comment (Dorthu) [reply src/data/endpoints/managed.yaml]: this should actaully be `phone_primray`

File: src/data/endpoints/managed.yaml (hunk 2 above)
Comment (Dorthu) [reply src/data/endpoints/managed.yaml]: this should actaully be `phone_secondary`

File: src/data/endpoints/managed.yaml [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+            type: String
+        examples:
+          curl: |
+            curl -H "Content-Type: application/json" \
+                -H "Authorization: Bearer $TOKEN" \
+                -X POST -d '{
+                    "name": "John Q. Public",
+                    "email": "johnqpublic@linode.com",
+                    "phone_primary": "215-555-3000",
+                    "phoen_secondary": "215-555-3001",
```
Comment (yoshizzle) [inline src/data/endpoints/managed.yaml]: doh!

Comment (SzySteve) [reply src/data/endpoints/managed.yaml]: you used to call me on my cell phoen

**Pull request merged**